BREAKING.md

```diff
@@ -24,6 +24,21 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 The `"batchEnd"` listener in `ISummarizerRuntime` has been deprecated and will be removed in a future release. Please remove all usage and implementations of `ISummarizerRuntime.on("batchEnd", ...)` and `ISummarizerRuntime.removeListener("batchEnd", ...)`.
 If these methods are needed, please refer to the `IContainerRuntimeBase` interface.
 
+### Deprecated events and event parameters on IContainer and IDeltaManager
+
+The following legacy events and event parameters have been marked as deprecated:
+
+- IContainerEvents
+    - "contextChanged": Event deprecated in its entirety. No replacement API recommended.
```

Comment:
Have we checked partner usage here? Please provide add short description reasoning around deprecation.

Reply:
I have confirmed no usages in the bohemia repo. Are there any other repositories I should explicitly check?

As for reasons / suggested alternative patterns, this is an area where I could use guidance / explanations. My current level of understanding for most of these is that they represent legacy patterns we don't want to support. I am not clear on what the alternatives are for many of them.

Reply:
I have expanded the comments a bit, but if there is more explicit guidance we can give, I am happy to update.